scripts/pi-hole/php/header.php

```diff
@@ -235,7 +235,7 @@ function pidofFTL()
             </a>
             <div class="navbar-custom-menu">
                 <ul class="nav navbar-nav">
-                    <!-- User Account: style can be found in dropdown.less -->
+                    <li class="dropdown"><a style="pointer-events:none;"><tt><?php echo gethostname(); ?></tt></a></li>
```

Comment:
Why not removing `class="dropdown"` to ensure that bootstrap does not add an eventhandler or something?
Furthermore, [the &lt;tt&gt;-Tag is not supported in html5](https://www.w3schools.com/tags/tag_tt.asp). So instead &lt;samp&gt; should be used.